### PR TITLE
Start at rending mars as well. 

### PR DESCRIPTION
**Pull request by dynosmurf**
I moved the rendering into two functions to allow earth and mars to be rendered on their own each time step. Need to parse the data for mars before the mars side will work though.

### CODE REVIEW

**Comment by dynosmurf** [comment]
I don't have any replays with units on mars yet, if you do could you add one to this repo for testing, if only in a test branch?

File: index.html [hunk 1]
```diff
@@ -78,16 +89,23 @@ <h1>bc18-tinyviewer</h1>
...
+    </div>
     <h2 id="winner"></h2>
 
 <script>
 
 var mousedown_listener, mouseup_listener, input_listener, needs_clear = false;
 
 var BORDER_WIDTH = 0.2;
-var canvas = document.getElementById('main');
+var canvas = document.getElementById('earth');
```
Comment (gabrielsimoes) [inline index.html:103]: IMHO this should be canvasEarth instead.

File: index.html [hunk 2]
```diff
@@ -78,16 +89,23 @@ <h1>bc18-tinyviewer</h1>
...
+var canvasMars = document.getElementById('mars');
+
 var ctx = canvas.getContext('2d');
```
Comment (gabrielsimoes) [inline index.html:106]: ...and ctxE

File: index.html [hunk 3]
```diff
@@ -224,11 +243,128 @@ <h2 id="winner"></h2>
...
+        document.getElementById('turnslider').value = (t - t % 4) / 4 + 1;
+        
+        // Render Karbonite reserves and turn number
+        document.getElementById('turn').innerText = document.getElementById('turnslider').value.toString();
+        document.getElementById('blue').innerText = reserves[t][0].toString();
+        document.getElementById('red').innerText = reserves[t][1].toString();
+
+    
+
+        render(t);
```
Comment (gabrielsimoes) [inline index.html:338]: ...and renderEarth

**Comment by gabrielsimoes** [comment]
I think you should be considering that Mars' dimensions can be different from earth's.

It would also be nice if you could make functions generic, instead of having two similar functions for Earth and Mars.

**Comment by dabbler0** [comment]
I'm going to close this pull request as a different implementation of the same feature is now in master.

**Pull request closed without merging**